src/Web/WebAPI/appsettings.Staging.json

```diff
@@ -25,5 +25,20 @@
   },
   "WarehouseGrpcClientOptions": {
     "BaseAddress": "http://warehouse-query"
+  },
+  "Serilog": {
+    "Using": [
+      "Serilog.Sinks.Elasticsearch"
+    ],
+    "WriteTo": [
+      {
+        "Name": "Console",
+        "Args": {
+          "Formatter": "Serilog.Formatting.Elasticsearch.ElasticsearchJsonFormatter,Serilog.Formatting.Elasticsearch",
+          "Theme": "Serilog.Sinks.SystemConsole.Themes.AnsiConsoleTheme::Code, Serilog.Sinks.Console",
+            "OutputTemplate": "{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz} [{Level:u3}] {Message:lj}{NewLine}{Exception}"
+        }
```

Comment:
```suggestion
       {
        "Name": "Console",
        "Args": {
          "Formatter": "Serilog.Formatting.Elasticsearch.ElasticsearchJsonFormatter,Serilog.Formatting.Elasticsearch",
          "Theme": "Serilog.Sinks.SystemConsole.Themes.AnsiConsoleTheme::Code, Serilog.Sinks.Console",
            "OutputTemplate": "{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz} [{Level:u3}] {Message:lj}{NewLine}{Exception}"
        }, {
        "Name": "Elasticsearch",
        "Args": {
          "nodeUris": "http://elasticsearch:9200"
        }
      }
```

Reply:
Thanks for the suggestion, but I'm looking for an agnostic approach. Where the application does not know about the logging provider. The idea is to set up filebeat to read the console and then send it to elastic